extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -35,6 +35,11 @@ const CONSENT_PROMPT_CAPTION = 'User Consent Prompt';
 const BUTTON_ACTION_CAPTION = 'Focus Prompt';
 const CANCEL_OVERLAY = 'cancelFullOverlay';
 const REQUEST_OVERLAY = 'requestFullOverlay';
+const ALLOWED_SANDBOX_ATTRIBUTES = [
+  'allow-popups-to-escape-sandbox',
+  'allow-top-navigation-by-user-activation',
+  'allow-top-navigation',
```

Comment:
talking to security, they do not want `allow-top-navigation`, the other 2 they have OK'd